Formula 1/ViewControllers/CircuitsCollectionView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CircuitsCollectionView.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import UIKit
+
+class CircuitsCollectionView: UICollectionView {
+    required init() {
+        let layout = ConstructorsFlowLayout()
+        
+        super.init(frame: UIScreen.main.bounds, collectionViewLayout: layout)
```

Comment:
This not a good idea. You should inject the frame that this should have if it needs one. Or try this, it may be perfectly fine:
```suggestion
        super.init(frame: .zero, collectionViewLayout: layout)
```

Reply:
it worked acere

Reply:
That's not a good answer. Doing the right thing is different from making it work until it doesn't.

Reply:
Sorry this was in reference to you telling me to try .zero. I mean that your suggestion works. Don't taze me bruh!